call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,115 @@
+/*Fix the following code: 
+	var obj = {
+		fullName: "Harry Potter",
+		person: {
+			sayHi: function(){
+				return "This person's name is " + this.fullName
+			}
+		}
+	}
+*/
+
+var obj = {
+	fullName: "Harry Potter",
+	sayHi: function(){
+		return "This person's name is " + this.fullName
+	}
+}
+
+//console.log(obj.sayHi());
+
+/*List two examples of 'array-like objects':
+	arguments (keyword w/in functions)
+	document.getElementsByClassName() -- many DOM methods return 
+		array-like objects
+	
+*/
+
+/*Write a function called sumEvenArguments which takes all of the arguments 
+passed to a function and returns the sum of the even ones.*/
+
+function sumEvenArguments(...args){
+	return args.reduce(function(acc, el){
+		if (el % 2 === 0) acc += el
```

Comment:
Thanks for this feedback! I will try to use these styling suggestions moving forward.